src/js/components/VoterGuide/OrganizationPositionItem.jsx

```diff
@@ -40,23 +42,10 @@ export default class OrganizationPositionItem extends Component {
             <span className="position-rating__candidate-name">{ballot_item_display_name}</span>
           </Link>
           
-          {/* wasn't sure how to get this to work - Jeff */}
           {/* show explicit position, if available, otherwise show rating */}
-          
-          {/* make into component */}
-          <div>See issue #278</div>
-          {/* if (position.is_support || position.is_oppose) {
-            org_position = <div className="explicit-position">
-              { position.is_support ? <img src="/img/global/icons/thumbs-up-color-icon.svg" width="20" height="20" className="explicit-position__icon" alt="Supports" /> : "[oppose icon]" }
-              <div className="explicit-position__text">
-                <span className="explicit-position__position-label">
-                  { position.is_support ? "Supports" : "Opposes" }
-                </span>
-                <span> {this.props.candidate_display_name}</span>
-              </div>
-            </div>);
-          } else if (vote_smart_rating) */}
-          <PositionRatingSnippet rating = {vote_smart_rating} rating_time_span = {vote_smart_time_span} />
+          { vote_smart_rating ?
```

Comment:
There should not be a vote_smart_rating when there is a support/oppose position. In general, I prefer to set up logic to look for cases that need specific data *first*, which is why I look for vote_smart_rating, and if not found, then look for the support/oppose/neutral position.